crates/widgets/src/numeric_box.rs

```diff
@@ -19,13 +17,13 @@ enum InputAction {
 }
 
 #[derive(Default, AsAny)]
-struct NumericBoxState {
-    action: Option<InputAction>,
+pub struct NumericBoxState {
```

Comment:
@kivimango why do it all public?

Reply:
In case of anyone needs some fields from the State struct in the callbacks where there is no Context available.

Reply:
Ok.